src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,68 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "implementation"(essentialLoader("gg.essential:loader-launchwrapper:1.1.3")!!)
+        }
+        tasks.named<Jar>("jar") {
+            dependsOn(essentialLoader)
+            manifest.attributes(mapOf(
+                "FMLCorePluginContainsMod" to "Yes, yes it does",
+                "TweakClass" to "gg.essential.loader.stage0.EssentialSetupTweaker"
+            ))
+            from({ zipTree(essentialLoader.singleFile) })
+        }
+    }
+
+    platform.isFabric -> {
+        dependencies {
+            "include"("modRuntimeOnly"("gg.essential:loader-fabric:1.0.0")!!)
+        }
+    }
+
+    platform.isModLauncher -> {
+        dependencies {
+            val relocatedPackage = findProperty("essential.loader.package")?.toString() ?: throw GradleException("""
+                A package for the Essential loader to be relocated to has not been set.
+                You need to set `essential.loader.package` in the project's `gradle.properties` file to a specific package where Essential's loader will be relocated to.
+                The recommended package is an unused package within your mod's package.
+            """.trimIndent())
```

Comment:
Would still add the example here.
```kotlin
              
              For example: `essential.loader.package = org.example.coolmod.relocated.essential`
```
(though afaict [from a quick search](https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html), properties files do not actually support quotes around the value, so I've removed those; their value always extends to the end of the line, so quotes are just not a thing the format has any need for)